src/components/MediatedRequestsActivities/components/MediatedRequestsFilters/components/MediatedRequestLevelFilter/MediatedRequestLevelFilter.test.js

```diff
@@ -1,21 +1,60 @@
+import { useIntl } from 'react-intl';
+
+import {
+  Accordion,
+} from '@folio/stripes/components';
+import {
+  CheckboxFilter,
+} from '@folio/stripes/smart-components';
```

Comment:
Lets use one approach for all imports:

```suggestion
import { useIntl } from 'react-intl';

import { Accordion } from '@folio/stripes/components';
import { CheckboxFilter } from '@folio/stripes/smart-components';
```

or 


```suggestion
import {
  useIntl,
} from 'react-intl';

import {
  Accordion,
} from '@folio/stripes/components';
import {
  CheckboxFilter,
} from '@folio/stripes/smart-components';
```

Reply:
Fixed